src/components/Dropdown.tsx

```diff
@@ -0,0 +1,78 @@
+import cn from 'classnames';
+import { Person } from '../types/Person';
+import { useRef, useState } from 'react';
+import '../App.scss';
+
+interface Props {
+  query: string;
+  onQueryChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
The naming convention for event handlers should be more descriptive and follow the pattern of 'handle<event>'. Consider renaming 'onQueryChange' to 'handleQueryChange' to make it clear that this is an event handler function.